core/editor/loader/editor-base-loader.php

```diff
@@ -250,6 +276,7 @@ public function register_additional_templates() {
 			'templates',
 			'navigator',
 			'hotkeys',
+			'responsive-bar',
```

Comment:
What about removing the template when app-bar is active?

Reply:
Done.